package.json

```diff
@@ -26,5 +26,8 @@
     "targz": "^1.0.1",
     "tempy": "^0.3.0",
     "test262-stream": "^1.3.0"
+  },
+  "devDependencies": {
+    "make-tap-output": "^2.0.0"
```

Comment:
Shouldn't it be a normal dependency?

Reply:
yes, while I fix that.... I put a screenshot